### ISSUE TITLE
Conda installation issue

### ISSUE THREAD

**Issue by jborbik** [avisingh599/reward-learning-rl#19]
I run into following problem when installing conda environment:

```
Running command git clone -q https://github.com/hartikainen/mujoco-py.git /tmp/pip-req-build-pv0fx_b7
  Running command git checkout -q 29fcd26290c9417aef0f82d0628d29fa0dbf0fab
  fatal: reference is not a tree: 29fcd26290c9417aef0f82d0628d29fa0dbf0fab
```

But replacing `git+https://github.com/hartikainen/mujoco-py.git@29fcd26290c9417aef0f82d0628d29fa0dbf0fab` with `git+https://github.com/hartikainen/mujoco-py.git` in requirements.txt solves the issue.

EDIT: But afterwards when running the example code I get:

```
/home/jedrzej/anaconda3/envs/softlearning/lib/python3.6/site-packages/requests/__init__.py:91: RequestsDependencyWarning: urllib3 (1.25.1) or chardet (3.0.4) doesn't match a supported version!
  RequestsDependencyWarning)
```
so it may be worth adjusting these dependencies.

**Comment by hartikainen** [comment]
Ah yeah, I just updated the master of my mujoco-py and haven't really tested against it.

The fix in `29fcd26290c9417aef0f82d0628d29fa0dbf0fab` should actually be incorporated in the latest `mujoco-py` upstream master. Could you try upgrading both gym and mujoco-py? I.e. `pip install -U mujoco-py gym`

**Comment by jborbik** [comment]
Ok, these two issues are related, I will close this one and continue in the other one
https://github.com/avisingh599/reward-learning-rl/issues/20
